chapters/connectors.tex

```diff
@@ -904,7 +904,16 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
 Before connection equations are generated, the virtual connection graph is transformed into a set of spanning trees by removing optional spanning tree edges from the graph.
-This is performed in the following way:
+
+Before that processing any redundant breakable branches are removed in each connection set:
+\begin{enumerate}
+\item
+  This is done by building a spanning tree within each connection set.
+\item
+  Breakable branches that are not part of any such spanning tree are redundant and removed.
```

Comment:
The _any such spanning tree_ sounds confusing to me.  Isn't it enough to only use the spanning tree built under the item above?
```suggestion
  Breakable branches that are not part of the spanning tree are redundant and removed.
  (The exact set of branches that are considered redundant is not defined, as it depends on the choice of spanning tree.)
```

Reply:
It depends on whether one considers all of the connection sets, or only one of them.
But I agree that focusing on one connection set makes it clearer.